build.sbt

```diff
@@ -27,10 +27,9 @@ val dependencies = Seq(
   "org.codehaus.janino"        % "janino"                       % "3.0.13" % Runtime,
   "com.github.pureconfig"      %% "pureconfig"                  % pureConfigVersion,
   "com.github.pureconfig"      %% "pureconfig-cats"             % pureConfigVersion,
-  "io.kamon"                   %% "kamon-prometheus"            % "1.1.2",
-  "io.kamon"                   %% "kamon-akka-2.5"              % "1.1.4",
-  "io.kamon"                   %% "kamon-core"                  % "1.1.6",
-  "io.kamon"                   %% "kamon-jmx-collector"         % "0.1.8",
+  "io.kamon"                   %% "kamon-bundle"                % "2.1.12",
+  "io.kamon"                   %% "kamon-prometheus"            % "2.1.12",
+  "io.kamon"                   %% "kamon-jmx-collector"         % "1.0.0-RC1",
```

Comment:
kamon bundle doesn't seem to contain prometheus, the build will fail without it.